js/crm.ajax.js

```diff
@@ -65,7 +65,7 @@
   CRM.api4 = function(entity, action, params, index) {
     return new Promise(function(resolve, reject) {
       if (typeof entity === 'string') {
-        $.post(CRM.url('civicrm/ajax/api4/' + entity + '/' + action), {
+        $.post(CRM.url(CRM.config.basePage + '/ajax/api4/' + entity + '/' + action), {
```

Comment:
I also don't understand why the downstream code calling `CRM.url()` would be expected to use `CRM.config.basePage`. I would expect that to be treated like an internal variable and in front-end contexts the function would automatically swap `'civicrm'` for `CRM.config.basePage`.
Otherwise, there is a mountain of core code and extensions throughout `civicrm-universe` to be audited for uses of `CRM.url()`...